ginga/rv/plugins/Preferences.py

```diff
@@ -268,6 +268,40 @@
 images are being loaded into a channel frequently (e.g., a low frequency
 video feed), it may be undesirable to create thumbnails for all of them.
 
+**General Preferences**
+
+The "Num Images" setting specifies how many images can be retained in
+buffers in this channel before being ejected.  A value of zero (0) means
+unlimited--images will never be ejected.  If an image was loaded from
+some accessible storage and it is ejected, it will automatically be
+reloaded if the image is revisited by navigating the channel.  A setting
+of 1 to 4 is typical.
+
+The "Sort Order" setting determines whether images are sorted in the
+channel alphabetically by name or by the time when they were loaded.
+This principally effects the order in which images are cycled when using
+the up/down "arrow" keys or buttons, and not necessarily how they are
+displayed in plugins like "Contents" or "Thumbs" (which generally have
+their own setting preference for ordering).
+
+The "Use scrollbars" check box controls whether the channel viewer will
+show scroll bars around the edge of the viewer frame.
+
+**Remember Preferences**
+
+When an image is loaded, a profile is created and attached to the image
+metadata in the channel.  These profiles are continuously updated with
+viewer state as the image is manipulated.  The "Remember" preferences
+control which parts of these profiles are restored to the viewer state
```

Comment:
Perhaps also an explanation of what would happen if restoring a previous state becomes impossible. For example, pan is saved for Image A, but it is an invalid pan (is such a thing possible?) for Image B, which is loaded instead the next time.

Reply:
So for your example, if you set a pan position in image A (lets say its a large one, and you pan out to the edge of it) and then view a new small image B, the new image does not have an existing profile. If the autocenter feature is turned to "off" then the pan position is just left where it was when you were viewing the large image and you may see nothing, until you press 'c' or center button in the toolbar. This is normal behavior without having anything to do with profiles.

With "restore pan" checked and you visit image B and center it, then go back to image A, you will be returned to the edge of the image A where you last had panned in it.  When you go back to B now, it will pan you to the center of it.

The point here is that the profiles don't change anything fundamentally with respect to having invalid settings. If you have autocuts turned off and you visit an image for which the existing cuts are invalid the image will usually look wrong.  The profiles can help in that if you constantly find yourself redoing some setting when going back and forth between images in a channel you might be able to avoid that by using some of the Remember settings.

This PR only changes things in that it adds the ability to restore the color map, color distribution and contrast changes (basically everything devoted to coloring the image) from the profile as well. So if you change the color map, or contrast for image A and then change it to something else for image B, then return to image A (*and* you have the "Restore color map" checkbox turned on) then it will restore the color settings exactly as they were when you were viewing that image.